src/identity/components/GlobalHeader/GlobalHeaderDropdown/index.tsx

```diff
@@ -153,7 +155,7 @@ export class GlobalHeaderDropdown extends React.Component<Props, State> {
     return (
       <Dropdown.Menu theme={dropdownMenuTheme} style={dropdownMenuStyle}>
         {/* Multi-org UI tickets #4051 and #4047, when user only has 1 account or 1 org, switch button is disabled */}
-        {typeAheadMenuOptions.length > 1 && (
+        {typeAheadOnly === false && typeAheadMenuOptions.length > 1 && (
```

Comment:
this is subtle, but `typeAheadOnly === false` rather than `!typeAheadOnly` implies that `typeAheadOnly` can be a falsy value (like '', or 0) but we only care if it's false. Is that the intent here?